cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/model/RandomCluster.java

```diff
@@ -37,6 +37,8 @@
  */
 public class RandomCluster {
 
+  public static final String TOPIC_WITH_ONE_LEADER_REPLICA_PER_BROKER = "TopicWithOneLeaderReplicaPerBroker";
+
```

Comment:
How is this topic used? Can we add some comment on its use?
Also 
Nit: `TopicWithOneLeaderReplicaPerBroker` --> `TopicWithOneLeaderPerBroker`